# Author: Caspar van Leeuwen
# Institute: SURF
# A 'jupyter serverextension enable --py jupyter_resource_usage' would fail, because
# _jupyter_server_extension_paths was not defined, and that's what notebook 6.4.0
# is looking for when loading extensions. This property got renamed in later versions
# of notebook. We define it here in the __init__ for backward compatibility.
# See https://github.com/jupyter-server/jupyter-resource-usage/issues/91
diff -Nru jupyter-resource-usage-0.6.1.orig/jupyter_resource_usage/__init__.py jupyter-resource-usage-0.6.1/jupyter_resource_usage/__init__.py
--- jupyter-resource-usage-0.6.1.orig/jupyter_resource_usage/__init__.py	2022-07-08 18:35:27.007610190 +0200
+++ jupyter-resource-usage-0.6.1/jupyter_resource_usage/__init__.py	2022-07-08 18:35:59.866559304 +0200
@@ -35,4 +35,6 @@
     ]
 
 
+# For backward compatibility
 load_jupyter_server_extension = load_jupyter_server_extension
+_jupyter_server_extension_paths = _jupyter_server_extension_points
